gradle.properties

```diff
@@ -5,13 +5,10 @@ android.minSdkVersion=21
 android.targetSdkVersion=30
 android.useAndroidX=true
 #
-# publish
-publish.groupId=jp.bucketeer
-publish.artifactId=sdk-android
-publish.vcsUrl=https://github.com/ca-dp/bucketeer-android-sdk
-#
 # com.vanniktech.maven.publish
-GROUP=jp.bucketeer
+GROUP=io.bucketeer
+# this VERSION_NAME is just a placeholder
+# actual VERSION_NAME is provided by CI
 VERSION_NAME=0.1.0-SNAPSHOT
```

Comment:
In publish-maven workflow, this value should be overridden by `ORG_GRADLE_PROJECT_VERSION_NAME` environment variable.